stockpile-wrapper/stockpile-wrapper.py

```diff
@@ -81,6 +90,10 @@ def main():
     parser.add_argument(
         '-p', '--port', 
         help='Provide elastic port information')
+    parser.add_argument(
+        '--usessl', 
```

Comment:
SSL is already supported by passing a _es_connection_string with the https schema. So I think we should call this variable somthing like `ssl_skip_verify`